Extensions/Multiplayer/multiplayertools.ts

```diff
@@ -941,6 +956,10 @@ namespace gdjs {
 
       // Handle message.
       switch (event.data.id) {
+        case 'listenerReady': {
```

Comment:
As discussed, probably something like `lobbiesViewLoaded` to make things clearer and avoid collisions in the future.